modules/core/src-2/Newtype.scala

```diff
@@ -24,6 +24,7 @@ abstract class Newtype[A] extends HasId { self =>
   type Type <: Base with _Tag
 
   @inline final def apply(a: A): Type = a.asInstanceOf[Type]
+  final val make: Newtype.Make[A, Type] = apply(_: A)
```

Comment:
With this change, is there ever a case where someone should use the apply method rather than make? If not, should the apply method be private?

Reply:
Users should very much keep using the apply method to construct instances of newtypes. This is very much a kind of hack to ensure we have a way of detecting whether a bijection is actually coming from a newtype.  I'll try to hide it further and make the `Make` construct package private.

Reply:
https://github.com/disneystreaming/smithy4s/pull/290/commits/e2d5e2e25e838715223b764858933e91f3349109